### PR TITLE
/settings UI

### LINKED ISSUE #180: /settings UI starting point.
Tentative UI for global /settings. Not to be confused with repository settings.

### PR DESCRIPTION
**Pull request by PeerRich**
Closes #180.

CleanShot 2023-03-12 at 21 52 05@2x

### CODE REVIEW

File: src/app/settings/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+];
+
+export default function SettingsLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  const { picture, initials, name } = useSession();
+  const metadata = useMetadata();
+  const pathname = usePathname();
```
Comment (PeerRich) [inline src/app/settings/layout.tsx:50]: I think I wanna load them on the server, not on the client.

maybe in a follow up PR if someone knows a better way

**Comment by cypherhoodlum** [comment]
Account and Profile might be the same thing in NostrGit?

**Comment by PeerRich** [comment]
i think i can ready for review this so we can add the subpages as standalone PRs

**Comment by PeerRich** [comment]
before adding too much forms we should look into react-hook-form to make our lives easier

**Comment by PeerRich** [comment]
> Account and Profile might be the same thing in NostrGit?

probably yeah!

**Comment by cypherhoodlum** [comment]
> unless account should be anything about nsec etc.

True, they could be on different tabs like on iris.to, but the naming should be different, I think.

**Review by cypherhoodlum — APPROVED** [review]
LGTM :+1: 
I'm done for today btw.

**Comment by e-roy** [comment]
> before adding too much forms we should look into react-hook-form to make our lives easier

plus one for react-hook-form!

**Review by e-roy — APPROVED** [review]

**Comment by PeerRich** [comment]
> > before adding too much forms we should look into react-hook-form to make our lives easier
> 
> 
> 
> plus one for react-hook-form!

@e-roy wanna open a PR adding it?

**Pull request merged**